src/main/java/com/spotinst/sdkjava/model/K8sVngClient.java

```diff
@@ -0,0 +1,133 @@
+package com.spotinst.sdkjava.model;
+
+import com.spotinst.sdkjava.exception.HttpError;
+import com.spotinst.sdkjava.exception.SpotinstHttpException;
+import com.spotinst.sdkjava.model.bl.ocean.kubernetes.*;
+import com.spotinst.sdkjava.model.requests.ocean.kubernetes.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class K8sVngClient {
```

Comment:
Rename K8sVngClient to SpotOceanK8sVngClient (or some meaningful name like this)